src/index.js

```diff
@@ -1,41 +1,15 @@
-import _ from 'lodash';
 import './style.css';
-import Icon from './icon.png';
-import addItem from './modules/addItem.js'
-
-
-  const todoList=[
-    {
-      descritpion: 'wash dishes',
-      completed:'false',
-      index: '0'
-    },
-    {
-      description: 'wash clothes',
-      completed:'false',
-      index: '1'
-    }
-  ];
-
-  const populateList = () => {
-   const parent= document.getElementById('populate');
-    for(let x=0; x< todoList.length; x +=1)
-    {
-      const ulList= document.
-      if (completed) 
-      {
-        const ulList= document.createElement('ul');
-        const liList=document.createElement('li');
-        liList.textContent=todoList[x].description;
-        ulList.appendChild(liList);
-        parent.appendChild(li);
-      }  
-        
-    }
-  }
-  
-  document.addEventListener('DOMContentLoaded', () =>{
-
-    populateList();
-  });
-  
\ No newline at end of file
+import {
+  todoList,
+  populateList,
+  curdFunctionality,
+  retrieveTodoListFromLocalStorage,
+} from './curd.js';
+
+document.addEventListener('DOMContentLoaded', populateList);
+document.addEventListener('DOMContentLoaded', () => {
+  const storedTodoList = retrieveTodoListFromLocalStorage();
+  todoList.push(...storedTodoList);
+
+  curdFunctionality();
+});
```

Comment:
The issue that is happening here is that currently when you create new todos, they are correctly being stored in local storage :heavy_check_mark: . However, the issue comes when you try reloading the page and now, the todos are no longer being displayed in your screen despite the fact that they are still intact in the local storage.


At the moment, the only way to see all the todos that are in the local Storage after you have reloaded the page is by creating another todo which will now prompt all the todos to be displayed because the ```populateList()``` is being called. This is not the desired output.

**Below is how I am seeing it just to demonstrate it better to you:**

[Screencast from 19-08-2023 09:59:46 ALASIRI.webm](https://github.com/umarahgul/webpackSetup/assets/86318284/907327e2-ad15-4fa1-8d41-fd3f3169c824)


**However, the fix is very simple. Have a look below:**

In your index.js file, you have the even listener for when the page is loaded and I realized that all the required functions are being called apart from one. The ```populateList()``` function! :bulb: . This means that by simply adding the ```populateList()``` right after the ```curdFunctionality()``` fixes this issue because you are getting the todos, then you also want to call the populate function to update your UI with the todos that are in the LocalStorage. I hope this makes sense to you?

**So now that I have solved the puzzle for you:**

- [ ] Please ensure that you add the missing function calls and ensure that it also works on your end as it does on one and once you are done please feel free to request another review in your dashboard.

Anyway, I also have a meme that fits your current situation enjoy :sweat_smile: 

![Uncalledfunc](https://github.com/umarahgul/webpackSetup/assets/86318284/7d2721e4-d863-4188-a841-ce5eb0941a71)